src/bem-cn/bem-names.js

```diff
@@ -33,7 +33,7 @@ export default function bemNames(entitys, delimiters) {
       /* eslint-disable no-param-reassign */
       if (val === true) {
         prev += ' ' + resultString + delims.mod + name;
-      } else if (isString(val)) {
+      } else if (isString(val) || typeof(val) === 'number') {
```

Comment:
Can you replace `number` detection with `isNumber` helper in [/src/utils.js](https://github.com/antiloopgmbh/vue-bem-cn/blob/master/src/utils.js) something like this:
```
const isNumber = val => typeof val === 'number' && isFinite(val);